src/components/Cart/CartItems.js

```diff
@@ -0,0 +1,109 @@
+import React from "react";
+import { useCart } from "../../context/cart-context";
+import { useWishlist } from "../../context/wishlist-context";
+import "./Cart.css";
+export const CartItems = () => {
+  const { cartState, cartDispatch } = useCart();
```

Comment:
Suggestion - Maybe you can use one dispatch function for both cart and wishlist, since both are interconnected.